### ISSUE TITLE
Draft SIP: SNX Community Fund Allocation Through Inflation

### ISSUE THREAD

**Issue by meanderson** [Synthetixio/SIPs#39]
### **TLDR;** 
As SNX begins to smooth current inflation rates, we believe that now is the best time to propose an ongoing-inflation treasury allocation to incentivize further growth and development of the Synthetix platform. We propose that this treasury be funded via a continuous allocation of weekly inflation, and that the funds accumulated be managed directly by the community through a multi-sig wallet. 

We propose naming this treasury vehicle the SNX Community Fund, and allocating 5-15% of weekly inflation towards it. 

We propose initiating a council, made up of team and community members, to govern and allocate the SNX Community Fund.

### **Background & Reasoning**
The changes that were introduced in [March 2019](https://blog.synthetix.io/synthetix-monetary-policy-changes/) spawned a fervent community of supporters of the Synthetix platform that have been driving growth for the last 7 months. As new challenges emerge, the team and community have proven their ability to alter the incentive structures, platform variables, and growth mechanics to subvert issues and continue the positive trajectory for Synthetix.

Over the last 7 months, we’ve also witnessed the massive success of discretionarily-allocated incentives for the Arb Contract, Uniswap sETH/ETH pool and Gitcoin Bounties. While some of these incentive systems have come from weekly inflation, others have been funded directly via the team/foundation. We believe there are substantial additional opportunities to build synth liquidity, acquire customers and drive network value if we can formalize the funding and allocation of similar growth initiatives. 

Institutionalizing an ongoing method of funding critical protocol initiatives, and building the muscle of governing these funds in an open way, are key to the SNX’s long-term success. By allocating these funds early on in the network’s life cycle while inflation is high, we give SNX the operating leverage to pursue network-accretive initiatives far down the road. By opening up the funds to be managed by the community, we take some of our first steps as a community towards truly decentralizing the platform; further engaging our stakeholders, mitigating regulatory risk, and increasing the level of trust and transparency held throughout the network. 

### **Proposed Model**
We believe that allocating a percentage of SNX inflation rewards into a community-operated fund is the best way to grow long-term network value. We propose that the SNX Community Fund tokens be housed in a multi-sig contract and controlled by a group of elected council members. This allocation of SNX should be independent of any of the existing incentive mechanisms (sETH pool and Arb Contract). 

Once the funds are in the multi-sig wallet, there needs to be a clearly defined process for what they can be used for, how decisions are made to fund proposals, and how to manage the ongoing treasury budget. We’ve detailed our proposed process for this in the next section. 

Below are a few options for fund allocations based on the proposed new inflation schedule in Deltatiger’s SIP. This proposal allocates the specified percentage inflation into a multi-sig wallet starting week 40 and includes a tail inflation of 100k SNX per week after year 4 (TBU with inflation change proposals). 

<img width="561" alt="Screenshot 2019-10-23 12 14 32" src="https://user-images.githubusercontent.com/761176/67426332-b9a0e000-f58e-11e9-8c37-c4386396c5d5.png">

<img width="632" alt="Screenshot 2019-10-23 12 14 41" src="https://user-images.githubusercontent.com/761176/67426344-befe2a80-f58e-11e9-9af4-eeab9c1339f3.png">

<img width="639" alt="Screenshot 2019-10-23 12 14 52" src="https://user-images.githubusercontent.com/761176/67426353-c4f40b80-f58e-11e9-9e2b-396a4e3c882f.png">

### **Treasury Governance**
Other blockchains use similar methods for governance and management of a treasury. Edgeware and Polkadot both implement a council of elected members to veto proposals, approve treasury expenditures and vote on proposals on behalf of network participants. While this model is likely too burdensome for our purposes, we can learn from their intentions and find something that fits our needs. 

We like how the Uniswap pool is managed by a multi-sig wallet and that think something similar, but with more specifically defined responsibilities (a formal council), could be a good starting place for how to govern the treasury.  

Tactically, we propose formalizing council-member responsibilities to be:

- Attending monthly community discussions where new funding (SIP) proposals are introduced
- Evaluating new (SIP) proposals and writing feedback on each (if any are proposed in a specified timeframe)
- Engaging with feedback from the community
- Deciding how much, if any, best fits the needs of the proposal
- Collaborating with other council members to unify potential options
- Make decision on how to move forward for each proposal

For the initial set of council members, we believe that anywhere from 5-10 is a good number of participants. We think that having more Synthetix team members with the majority of seats at the start makes the most sense, but eventually transitioning to more independent community members. We also think the best model is to have 6-month terms, and elections voted on by the community (in Discord).

### **Reasons For This Model**
We believe this model for a treasury funded through an allocation of the weekly inflation would accomplish the goal of allowing the platform to iterate and experiment without facing roadblocks during ad-hoc requests, as well as showcase decentralized governance to build trust and transparency within the community.

Some of the future use cases that the treasury could be used to fund:

- Referral program for sX users
- Synth-specific liquidity pools (sUSD/USDC Uniswap pool)
- Larger Gitcoin bug bounties
- Community-driven feature development

### **Reasons Against This Model**
The core reasons to be against this model are that:
Lack of belief that growth has been incentivized through specific mechanisms (i.e. Uniswap sETH pool LP rewards)
Lack of belief that the Synthetix network will continue to face growth issues
Belief that Synthetix Foundation should fund any future growth mechanisms
Belief that there should be a treasury, but that it should be controlled by the Synthetix team
Belief that an undefined governance model is preferable to a defined model
Belief that an on-chain governance model is preferable to a council-based model

### **Open Decisions**
The open questions that need decisions are the following:

- Should there be a treasury?
- How should this treasury be funded? 
- How should the treasury be managed?
- If there is a council, how do we elect people to it?
- If there is a council, how many people should be on it?
- If there is a council, how long should the terms be?
- If there is a council, what quorum is needed to allocate to an initiative?

**Comment by CryptoToit** [comment]
Some interesting discussions will come from this SIP. 

As far as I understand, there is already a Foundation with funds to support the Team and 'pay' for certain things. Before a new Treasury is founded I would like to see some kind of high level audit on the current funds available and how it is allocated to a) operations, b) development and c) incentives. 

Should a separate / new Treasury be set up or should the current one not maybe be converted / upgraded to a more decentralise style? 

I like the idea, but personally I don't want to rush this as the Team is doing a good job and we should not implement something that could negatively affect the momentum that has been build up to date. There are still a lot to be done before we really hit version 1 of the entire new system. 

But good to start the discussion and get some high IQ input from the community and the Team.

**Comment by FrameworkVance** [comment]
> Before a new Treasury is founded I would like to see some kind of high level audit on the current funds available and how it is allocated to a) operations, b) development and c) incentives.

Resolving this is a reasonable first step and I think is imperative to making an informed decision.

**Comment by Clariti23** [comment]
Thank you Mike and Vance, for this very well-thought out Draft SIP! 

To the SNX Community: 
 
****TL;DR: **

I am FOR experimenting with a Council of a dozen individuals voted on by the community with an allocation on the modest end of the spectrum (5-10%).  

I am FOR specifying a limited scope for the Council to execute on: Growth Initiatives.** 

It seems pretty clear that the focus of the council outlined here is growth initiatives. Tasking the Council with executing critical protocol initiatives is such a wide-open goal, how we could know the council had achieved its raison d'être, at least in a minimally abstract sense? Wen $6 dolla perhaps? 

The Arb Pool resolved a specific need for a healthier peg; the Uniswap Pool Rewards resolved a specific need for on/off-ramp liquidity. A more specific definition of the council’s goal might be to accelerate growth in the # of users participating with the network. Once the network has reached some self-evident point where user growth has adequately accelerated, the council might then evolve to tackling broader agenda items?

I am inclined to support a trial phase experiment how well the council works in practice before establishing the council as an ingrained mechanism of the protocol. Then there is time to generate data on how this form of governance looks and acts in practice.

While it’s not aesthetically pleasing to run with an undefined governance model, part of me wonders if we should stick with the mantra that if something isn’t broke, don’t fix it. It's a very small part of me, though.

“If all you have is a hammer, everything looks like a nail.” One concern about institutionalizing SNX governance is the Council will find projects that are more 'wants' than 'needs', because the Council has funds and the Council exists in part to distribute funds. That said, I appreciate the Council’s raison d'être is being dynamic in relation to a fast-evolving DeFi environment. Keeping the Council’s aims specific also quells my concern on this front. 

The team has no shortage of protocol initiatives in the works, so I think it’s worth asking Kain and members of the team how much should realistically be expected of them in terms of their contribution? 

**Tactically:** 

IF the Council is activated, I propose the Council reach consensus on appropriation initiatives with 3/4 majority vote (9 of 12). 

IF the Council is activated, I propose a quorum of 9 members, with 6 votes being required to appropriate funds toward an initiative. 

IF the Council is activated, I propose one of the Council members role for a secretary to take minutes at Council meetings that can be shared with the community, to encourage transparency. Alternatively, the meetings can be streamed, recorded, and posted on YouTube by the secretary.

IF the Council is activated, I propose community votes for Council members be held on the SNX Discord within a 24-hour period of a governance call. 

If the Council is activated, I propose allowing for self-nomination as well as allowing community-nominations of candidates for Council member positions. 

If the Council is activated, I propose evaluating the composition, existential status, and performance of the Council with a set of metrics that the community agrees on as being a valid benchmarks to measure how well the Council is achieving its intended purpose. 
 
If the Council is activated, I propose a formal community referendum be held in 6 or 12 months after activation on Discord to decide whether to keep or modify the Council. 

_Open Question: If there is agreement that Growth is the proper scope of the Council, how will we make this specific goal concrete, i.e. what metrics/KPIs should we focus on when thinking about growth?_

**Comment by meanderson** [comment]
Following the governance call today, I want to provide a quick review of the discussion and then the next steps.

One of the guiding questions that has come up in the comments, as well as the discord, is the need for clarity on the desired purpose for the Foundation SNX funds. This answer will dictate the best path towards funding a Community Fund, either through these pre-allocated funds or as a % of ongoing inflation. @kaiynne confirmed that the Foundation SNX funds are to be thought of as "belonging to the network". In this regard, we think it's redundant to take a % of inflation to dedicate to a fund seeing as they already exist. Therefore let's consider that part of this discussion settled.

The second part of this WIP SIP has to do with the management and distribution of the allocated funds. This section is still relevant as we discussed, a few times throughout the call, the benefits of furthering the level of decentralization throughout the platform, both technical and non-technical. From a governance perspective, we discussed the negative perspective of on-chain governance and some of the potential virtues of a pseudo-representative-democracy model (potentially even a council). There are a ton of open questions that come with any model (a few mentioned above), and even small differences between each model could cause major tradeoffs when implemented. It's clear that this is not something that will be solved 1) soon, and 2) without experimentation, but it seems that there was a general agreement on a desire to get the ball rolling for trying these governance models at a small scale.

As the next steps, @FrameworkVance and I can do further research on a few different models for governance and come back with our findings. We'll leave this WIP SIP here as a starting spot and open forum for others' opinions on potential decentralized governance models. By the next governance call, our goal will be to have a better idea of potential governance models, examples in use today, and potential trade-offs between different implementations. At that point, we can decide if we want to move forward drafting a different SIP to potentially test a model.
